sdk/python/jobs/pipelines/2b_train_cifar_10_with_pytorch/train-model/main.py

```diff
@@ -15,9 +15,11 @@
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
+import numpy as np
+import copy
+#- add mlflow logging
 
-# TODO - add mlflow logging
-
+mlflow.set_experiment("pipeline_samples")
```

Comment:
nit: may want to change spacing like

# add mlflow logging
mlflow.set_experiment(...)

and then move both these lines inside main function. also let me see if there's a way to avoid using hard-coded experiment string here